src/test/java/persistence/entity/EntityLoaderTest.java

```diff
@@ -88,20 +88,105 @@ void entity() {
         jdbcTemplate.execute("insert into order_items (id, product, quantity, order_id) values (2, 'product2', 2, 1)");
```

Comment:
```suggestion
        jdbcTemplate.execute("insert into order_items (id, product, quantity, order_id) values (2, 'product2', 2, 1)");
        jdbcTemplate.execute("insert into orders (id, order_number) values (2, 'order2')");
        jdbcTemplate.execute("insert into order_items (id, product, quantity, order_id) values (3, 'product3', 2, 2)");
```
위 조건을 추가하니 test case 가 실패하네요🥲

Reply:
위 코멘트를 반영하니 해결되었네요:)